src/container.ts

```diff
@@ -494,6 +505,19 @@ export function createContainer(
     }
   }
 
+  /**
+   * Checks if the registration with the given name exists.
+   *
+   * @param {string | symbol} name
+   * The name of the registration to resolve.
+   *
+   * @return {boolean}
+   * Whether or not the registration exists.
+   */
+  function has(name: string | symbol): boolean {
+    return name in registrations;
```

Comment:
You need to do `name in rollUpRegistrations()`, otherwise parent container registrations won't be picked up.
```suggestion
    return name in rollUpRegistrations()
```